### PR TITLE
Myfatoorah icons

### PR DESCRIPTION
**Pull request by nshoman-mf**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by nshoman-mf** [comment]
Adding MyFatoorah Payment Gateway icons,
Kfast, Knet, Sadad, Mada, Naps, benefit and UAE visa

**Comment by nshoman-mf** [comment]
What is the problem with this build?

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    payment_icons (1.4.4)
+    payment_icons (1.4.5)
```
Comment (tauthomas01) [inline Gemfile.lock:4]: Can you please revert this file? I will be the one bumping the version.

Comment (nshoman-mf) [reply Gemfile.lock:4]: Done

**Comment by nshoman-mf** [comment]
> Hi @nshoman-mf, I have a few comments about your changes.
> 
> 1- Please do not use style tags inside the SVG file. Instead, you can use the fill or stroke attribute directly to the element.
> 2- All of your icons need to have a border around the icon to keep consistency. Here is an [example](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg).
> 
> For more informations, please look at the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#template) guideline.

Please check now..

**Comment by nshoman-mf** [comment]
> Thank you for following the contributions guidelines.

Thank you so much.
Can I know, when will it be published in Shopify partner - payment types?

**Comment by tauthomas01** [comment]
It will be available beginning of next week.

**Comment by nshoman-mf** [comment]
> It will be available beginning of next week.

Hello, I know our icons are published in ShipIt, but when will icons be released by Shopify partner Gateway - Payment Type ?

**Pull request merged**